engines/kinduction.cpp

```diff
@@ -356,6 +356,11 @@ bool KInduction::check_simple_path_lazy(int i)
     // we return immediately here and skip simple path checking.
     if (options_.kind_no_simple_path_check_) return false;
 
+    if (r.is_unknown()) {
+      kind_log_msg(2, "      ", "simple path check UNKNOWN");
+      return false;
+    }
```

Comment:
I would use `!r.is_sat()` to catch also `r.is_null()` (though it is unlikely to happen):

```suggestion
    if (!r.is_sat()) {
      kind_log_msg(2, "      ", "simple path check {}", r.to_string());
      return false;
    }
```